File: 0237-delete-node-in-a-linked-list/0237-delete-node-in-a-linked-list.java
/**
 * Definition for singly-linked list.
 * public class ListNode {
 *     int val;
 *     ListNode next;
 *     ListNode(int x) { val = x; }
 * }
 */
class Solution {
    public void deleteNode(ListNode node) {
      
        ListNode temp=node;
        ListNode temp1=node.next;
        
        temp.val=temp1.val;
        temp.next=temp1.next;
    }
}

File: 0237-delete-node-in-a-linked-list/0237-delete-node-in-a-linked-list.cpp
/**
 * Definition for singly-linked list.
 * struct ListNode {
 *     int val;
 *     ListNode *next;
 *     ListNode(int x) : val(x), next(NULL) {}
 * };
 */
class Solution {
public:
    void deleteNode(ListNode* node) {
        ListNode *ptr=node;
        ListNode *ptr1=node->next;

       ptr->val=ptr1->val;
       ptr->next=ptr1->next;
    }
};